docs/dev/replay-exec.md

```diff
@@ -0,0 +1,82 @@
+This doc describes how to dump an Exec runtime meta and column batch data to a directory,
+and replay the Exec with the dumped meta and data. When encountering a perf issue using this tool
+can dump the runtime meta/data and then replay. It will spend less time when collecting NSYS and 
+NCU information when replaying the dumped runtime meta/data.
+
+# Dump a Exec meta and a column batch data
+
+## compile Spark-Rapids jar
+e.g.: compile for Spark330
+```
+mvn clean install -DskipTests -pl dist -am -DallowConventionalDistJar=true -Dbuildver=330 
+```
+
+## enable dump
+This assumes that the RAPIDs Accelerator has already been enabled.   
+This feature is disabled by default.   
+Set the following configurations to enable this feature:   
+
+``` 
+spark.conf.set("spark.rapids.sql.test.replay.exec.type", "project")
```

Comment:
why is there a ".test." in the name of this config?  isn't this for debugging, testing makes it sound not to be used in real workloads and for testing purposes

Reply:
Now updated to `.debug.`